sql/sql_base.cc

```diff
@@ -1001,9 +1001,6 @@ void close_thread_table(THD *thd, TABLE **table_ptr)
     table->file->ha_reset();
   }
 
-  /*
```

Comment:
will we have a new comment?

Reply:
There seems to be no need for a comment

Reply:
I mean [the one for the declaration](https://github.com/MariaDB/server/pull/2680#discussion_r1276456880)

Reply:
i added this

Reply:
thanks, missed it.